prisma/schema.prisma

```diff
@@ -112,6 +112,7 @@ model Volunteer {
   criminalRecordCheckLink String?        @map("criminal_record_check_link")
   criminalCheckApproved   Boolean?       @map("criminal_check_approved")
   criminalCheckExpired    Boolean?       @default(false) @map("criminal_check_expired")
+  criminalCheckSubmittedAt DateTime?     @map("criminal_check_submitted_at")
```

Comment:
let's make sure these timestamp are the same length as the createdAt/updatedAt columns
```suggestion
  criminalCheckSubmittedAt DateTime?     @map("criminal_check_submitted_at") @db.Timestamptz(6)
```